### ISSUE TITLE
Restart on voice download

### ISSUE THREAD

**Issue by krishnshyam** [Hear2Read/hear2read-ng-nvda#5]
Restarting NVDA after voice download and install some times fails with the error:
```
ERROR - core.triggerNVDAExit (20:13:02.423) - MainThread (10376):
NVDA cannot exit safely, ensure open dialogs are closed
ERROR - core.restart (20:13:02.442) - MainThread (10376):
NVDA already in process of exiting, this indicates a logic error.
ERROR - unhandled exception (20:13:02.945) - MainThread (10376):
Traceback (most recent call last):
  File "wx\core.pyc", line 3425, in <lambda>
  File "gui\__init__.pyc", line 946, in run
  File "gui\message.pyc", line 64, in funcWrapper
  File "gui\message.pyc", line 97, in displayDialogAsModal
RuntimeError: wrapped C/C++ object of type Hear2ReadNGVoiceManagerDialog has been deleted
INFO - core.main (20:13:11.069) - MainThread (10376):
Exiting
```

**Comment by krishnshyam** [comment]
Corrected in 1ba6d7220bee31679403ad0a898c0b06b79a38db
